charts/sn-platform/templates/presto/presto-worker-deployment.yaml

```diff
@@ -101,7 +95,7 @@ spec:
             - >-
               cp {{ template "pulsar.home" . }}/conf/presto/node.properties.template {{ template "pulsar.home" . }}/conf/presto/node.properties;
               echo "node.id=${HOSTNAME}" >> {{ template "pulsar.home" . }}/conf/presto/node.properties ;
-              echo "node.internal-address=${HOSTNAME}.{{ template "presto.worker.service" . }}.{{ template "pulsar.namespace" . }}.svc.cluster.local" >> {{ template "pulsar.home" . }}/conf/presto/node.properties ;
+              echo "node.internal-address=${POD_IP}" >> {{ template "pulsar.home" . }}/conf/presto/node.properties ;
```

Comment:
It works for non-Istio env, but it will break mTLS and fail in Istio env like what happens in Prometheus.